lib/dal-test/src/schemas/test_exclusive_schema_katy_perry.rs

```diff
@@ -0,0 +1,128 @@
+    let string_codegen_fn_name = "test:generateStringCode";
+    let string_codegen_func_code = "async function main(input: Input): Promise < Output > {
+                return {
+                    format: \"string\",
+                    code: \"poop canoe\"
+                };
+            }";
+    let string_code_gen_func =
+        build_codegen_func(string_codegen_func_code, string_codegen_fn_name).await?;
+
+    let kp_schema = SchemaSpec::builder()
+        .name("katy perry")
+        .data(
+            SchemaSpecData::builder()
+                .name("katy perry")
+                .category("test exclusive")
+                .category_name("katy perry")
+                .build()
+                .expect("schema spec data build"),
+        )
+        .variant(
+            SchemaVariantSpec::builder()
+                .name("v0")
+                .unique_id("katy_perry_sv")
+                .data(
+                    SchemaVariantSpecData::builder()
+                        .name("v0")
+                        .color("#ffffff")
+                        .func_unique_id(&kp_authoring_schema_func.unique_id)
+                        .component_type(ComponentType::Component)
+                        .build()
+                        .expect("build variant spec data"),
+                )
+                .domain_prop(
+                    PropSpec::builder()
+                        .name("name")
+                        .kind(PropKind::String)
+                        .func_unique_id(&identity_func_spec.unique_id)
+                        .input(
+                            AttrFuncInputSpec::builder()
+                                .kind(AttrFuncInputSpecKind::Prop)
+                                .name("identity")
+                                .prop_path(PropPath::new(["root", "si", "name"]))
+                                .build()?,
+                        )
+                        .build()?,
+                )
+                .leaf_function(
+                    LeafFunctionSpec::builder()
+                        .func_unique_id(yaml_codegen_fn_name)
+                        .leaf_kind(LeafKind::CodeGeneration)
+                        .inputs(vec![LeafInputLocation::Domain])
+                        .build()?,
+                )
+                .leaf_function(
+                    LeafFunctionSpec::builder()
+                        .func_unique_id(string_codegen_fn_name)
+                        .leaf_kind(LeafKind::CodeGeneration)
+                        .inputs(vec![LeafInputLocation::Domain])
+                        .build()?,
+                )
+                .build()?,
+        )
+        .build()?;
+
+    let swifty_spec = kp_builder
+        .func(identity_func_spec)
+        .func(resource_payload_to_value_func)
+        .func(yaml_code_gen_func)
+        .func(string_code_gen_func)
+        .schema(kp_schema)
+        .build()?;
+
+    let swifty_pkg = SiPkg::load_from_spec(swifty_spec)?;
```

Comment:
Is she Taylor or Katy?? :exploding_head:

Reply:
d'oh!! good catch on the naming - fixing now :)